apollo-federation/src/sources/connect/models/test_data/validation/invalid_connect_http_headers.graphql

```diff
@@ -0,0 +1,19 @@
+extend schema
+  @link(url: "https://specs.apollo.dev/connect/v0.1", import: ["@connect"])
+
+type Query {
+  resources: [String!]!
+    @connect(
+      http: {
+        GET: "http://127.0.0.1:8000/resources"
+        headers: [
+          { name: "Accept", value: "text/html" }
+          { name: "Accept", as: "Accept" }
+          { as: "Accept", value: "text/html" }
```

Comment:
I think `name` is required, so this will eventually fail GraphQL validation when we start checking it.